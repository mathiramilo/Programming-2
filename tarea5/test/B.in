hacerVecinos 1 2 1.0
sonVecinos 1 2
sonVecinos 3 4
distancia 1 2
vecinos 1
vecinos 2
vecinos 4
hayMParejas
hacerVecinos 1 4 1.0
hacerVecinos 3 2 1.0
vecinos 1
vecinos 2
vecinos 4
hayMParejas
accesibles 3
accesibles 10
Fin
