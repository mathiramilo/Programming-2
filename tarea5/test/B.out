1>Nuevos vecinos: 1 2 1.00.
2>Son vecinos.
3>NO son vecinos.
4>1.00
5>2 
6>1 
7>
8>NO hay M parejas.
9>Nuevos vecinos: 1 4 1.00.
10>Nuevos vecinos: 3 2 1.00.
11>2 4 
12>1 3 
13>1 
14>NO hay M parejas.
15>1 1 1 1 0 0 0 0 0 0 
16>0 0 0 0 0 0 0 0 0 1 
17>Fin.
